editorconfig-maven-plugin/src/main/java/org/ec4j/maven/AbstractEditorconfigMojo.java

```diff
@@ -219,6 +219,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         } else {
             this.charset = Charsets.forName(this.encoding);
         }
+        if (basedir == null) {
+            basedir = new File(".").getAbsoluteFile();
```

Comment:
@ppalaga @qxo I was wondering if it is ever possible for `basedir` to be `null` here. Also, I think Maven itself uses something like `System.getProperty( "user.dir" )` and not `new File(".").getAbsoluteFile()`.

Reply:
It is easy to figure out by writing a simple test :)

Reply:
sorry the delay, we run this plugin goal on none-maven project folder, the basedir is null